tests/Stubs/Bridge/Symfony/MessageBusStub.php

```diff
@@ -0,0 +1,19 @@
+<?php
+declare(strict_types=1);
+
+namespace Tests\LoyaltyCorp\Search\Stubs\Bridge\Symfony;
+
+use Eonx\TestUtils\Stubs\BaseStub;
+use Symfony\Component\Messenger\Envelope;
+use Symfony\Component\Messenger\MessageBusInterface;
+
+final class MessageBusStub extends BaseStub implements MessageBusInterface
```

Comment:
MessageBusInterface doesnt exist?

Since we havent migrated to github actions here yet and bamboo doesnt work on forks, can you please ensure to run `vendor/bin/es` locally? We should have actions across these repos soon

Reply:
Looks like it might be a missing package dependency? symfony/messenger is not listed in composer.json

Reply:
Yes, I did run the `vendor/bin/es -f` locally and it passed.

Reply:
@merk The `MessageBusInterface` exist in `symfony/symfony` package, might be you have not done `composer update` on your local to install symfony?

Also I noticed, `vendor/bin/es` returns errors for phpstan but `vendor/bin/es -f` (full scan) does not?

Reply:
Sorry, I only ended up running a composer install which isnt going to do it.

Hopefully we can have github actions soon :)

Reply:
Alright. Thanks! :)